app/src/app/components/files/files.component.ts

```diff
@@ -115,6 +117,15 @@ export class FilesComponent extends PageComponent implements OnInit, OnDestroy {
       });
 
     this.columns = this.extensions.documentListPresets.files || [];
+
+    this.discoveryApiService
```

Comment:
this should not be in ACA as AGS is part of the ADW... what exactly you are trying to achieve? hiding the "name" column?